### ISSUE TITLE
New bug report

### ISSUE THREAD

**Issue by jlamasfripp** [meetalva/alva#788]
Hey there, I just encountered the following error with Alva:
It says Material Design is not compatible.

<details><summary>Stack Trace</summary>

```
Error
    at getBundle (/Applications/Alva Canary.app/Contents/Resources/app.asar/index.js:206141:23)
    at Object.analyze (/Applications/Alva Canary.app/Contents/Resources/app.asar/index.js:206154:25)
    at /Applications/Alva Canary.app/Contents/Resources/app.asar/index.js:82376:32
```

</details>

**Comment by yuzl** [comment]
#783

**Comment by yuzl** [comment]
Thanks for your report. We will let you know as soon as we fixed install bug as reported here #783
